_posts/2022-03-25-dyadic-fairness.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "On Dyadic Fairness: Exploring and Mitigating Bias in Graph Connections"
+tags: [graphs, fairness, representation learning]  # This should be the relevant areas related to your blog post
+authors: Arjun Subramonian
```

Comment:
Can you switch your author name to follow the format:

`last_name, firstname`? 

Its important for the bibtex generation :)